postgraas_server/management_resources.py

```diff
@@ -86,14 +86,27 @@ def delete(self, id):
                     host=current_app.postgraas_backend.master_hostname,
                     port=entity.port,
                     dbname=entity.db_name
-            ):
+            ) as conn:
                 pass
         except Exception as ex:
             return_code = 401 if 'authentication failed' in str(ex) else 500
             abort(return_code, status='failed',
                   msg='Could not connect to postgres instance: {}'.format(str(ex))
                   )
 
+        try:
```

Comment:
Something is off with the indention here. As the `with` block ended the context manager should actually already have closed the connection.